File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/fingo_operation/ProcessingFinished.java
package com.yelloco.fingodriver.models.fingo_operation;

import com.yelloco.fingodriver.enums.FingoErrorCode;

public class ProcessingFinished
{
    private boolean status;
    private int errorCode;
    private String text;
    private String errorName;

    public ProcessingFinished(){
    }

    public boolean getStatus() {
        return status;
    }

    public void setStatus(boolean status) {
        this.status = status;
    }

    public int getErrorCode() {
        return errorCode;
    }

    public void setErrorCode(int errorCode) {
        this.errorCode = errorCode;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getErrorName() {
        return errorName;
    }

    public void setErrorName(String errorName) {
        this.errorName = errorName;
    }

    @Override
    public String toString() {
        return "ProcessingFinished{" +
                "status=" + status +
                ", errorCode=" + errorCode +
                ", text='" + text + '\'' +
                ", errorName='" + errorName + '\'' +
                '}';
    }
}

File: FingoDriver/src/main/java/com/yelloco/fingodriver/models/fingo_operation/ProcessingFinished.kt
package com.yelloco.fingodriver.models.fingo_operation

class ProcessingFinished
{
    var status = false
    var errorCode = 0
    var text: String? = null
    var errorName: String? = null

    override fun toString(): String {
        return "ProcessingFinished{" +
                "status=" + status +
                ", errorCode=" + errorCode +
                ", text='" + text + '\'' +
                ", errorName='" + errorName + '\'' +
                '}'
    }
}